app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { response } = require('express');
+
+const app = express();
+app.use(express.static('public'))
+
+app.get('/home',(request,response,next)=>{
+    response.sendFile(__dirname+'/views/home.html')
+})
```

Comment:
Esta forma está bien, pero mejor que utilicéis esta otra, ya que con render puedes pasar parámetros :

app.get('/', (req, res, next) => res.render('home'))

También res, req, next, es igual de legible y lo verás siempre así.
En el ejemplo la home está en '/', que suele ser lo habitual.